app/src/main/res/layout/activity_button.xml

```diff
@@ -0,0 +1,177 @@
+        android:id="@+id/app_bar_container"
+        layout="@layout/app_bar_demo"/>
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_medium"
+        android:layout_marginEnd="@dimen/margin_medium"
+        app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/text_view_span_title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Title"
+                tools:ignore="HardcodedText" />
+            <TextView
+                android:id="@+id/text_view_span_usage"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Usage"
+                tools:ignore="HardcodedText" />
+            <TextView
+                android:id="@+id/text_view_span_anatomy"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Anatomy"
+                tools:ignore="HardcodedText" />
+            <TextView
+                android:id="@+id/text_view_span_hierarchy"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Hierarchy"
+                tools:ignore="HardcodedText" />
+            <TextView
+                android:id="@+id/text_view_span_placement"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Placement"
+                tools:ignore="HardcodedText" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content">
+                <TextView
+                    android:id="@+id/text_view_span_text_button_usage"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="Text button usage"
+                    tools:ignore="HardcodedText,MissingConstraints" />
+
+                <Button
+                    android:id="@+id/textButton"
+                    style="@style/Widget.MaterialComponents.Button.TextButton.Icon"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Text button"
+                    app:icon="@drawable/ic_search_24dp"
```

Comment:
Import a plus icon from the android vector asset and use it instead of the search icon, do it if you like it, it is not a big deal anyway.